### PR TITLE
Starting project

### PR DESCRIPTION
**Pull request by designatednerd**
Note: Leaving the boilerplate from Apple in so I can work with it to figure out WTF is going on here.

### CODE REVIEW

**Comment by designatednerd** [comment]
Hey @vokal/ios-developers Does this look like a project?

File: SwiftSnake/SwiftSnakeTests/SwiftSnakeTests.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//
+
+import XCTest
+@testable import SwiftSnake
+
+class SwiftSnakeTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        XCTFail("Y U NO WRITE TESTS?")
```
Comment (chillpop) [inline SwiftSnake/SwiftSnakeTests/SwiftSnakeTests.swift:16]: if you care about boilerplate, you got some in here

**Comment by chillpop** [comment]
aside from the template boilerplate LGTM

File: SwiftSnake/SwiftSnakeTests/SwiftSnakeTests.swift (hunk 1 above)
Comment (designatednerd) [inline SwiftSnake/SwiftSnakeTests/SwiftSnakeTests.swift:16]: This is my reminder to myself to write tests :smile:

**Comment by designatednerd** [comment]
Yeah, a bunch of that will come out in the next pull.

**Comment by brockboland** [comment]
Code review? THIS IS HACK DAYS

**Comment by bryanluby** [comment]
@brockboland :star:

**Pull request merged**